library/src/main/java/com/google/maps/android/clustering/view/DefaultClusterRenderer.java

```diff
@@ -225,7 +226,13 @@ public int getColor(int clusterSize) {
         });
     }
 
-    @NonNull
```

Comment:
May have been overwritten when resolving merge conflict. Should this be @NonNull too?

Reply:
Not, it is not really overwritten. This belongs to `getClusterText()`, which still keeps the `@NonNull` annotation. It is just a weird formatting from GitHub in the diff.